### DISCUSSION TITLE
Feature request: Add DBT Jinja function which returns the DBT CLI command used to launch the current invocation.

### DISCUSSION THREAD

**Discussion by Yttriums** [dbt-labs/dbt-core#6183] [Ideas]
Hi all,
Wanted to request the addition of a new dbt Jinja command (similar to invocation_id and selected_resources) that returns the CLI command used to launch the current invocation.

Not sure how feasible/widely needed this is, but it would be a great addition to invocation-level audit tables (especially for production environments in orgs where storing this information is a standard). Apologies if there's another way to fetch this value that I'm missing. Thanks for all of the amazing work 🙏

**Comment by jtcohen6** [comment]
This was just opened as an issue! https://github.com/dbt-labs/dbt-core/issues/6051

**Reply by Yttriums** [reply]
Wonderful news. This is going to enable some truly goofy macros on my part. Go ahead and close this discussion if it's redundant.🙂
